lib/sensu/settings/validators/client.rb

```diff
@@ -110,6 +110,8 @@ def validate_client(client)
               invalid(client, "client name cannot contain spaces or special characters")
             must_be_a_string(client[:address]) ||
               invalid(client, "client address must be a string")
+            must_be_a_string_if_set(client[:client_key]) ||
```

Comment:
Hey @portertech. I opted for `client_key` as `key` seemed a bit too ambiguous. Naming things is always hard. `client_key` tells you it's a client-side value as opposed to some authentication key that the server would require. Naming things is always hard...